src/pages/dashboard/index.js

```diff
@@ -39,7 +39,7 @@ const Dashboard = () => {
 					</div>
 				</Card>
 
-				<Posts />
+				{/* <Posts /> */}
```

Comment:
Let's uncomment this. A fix has been merged into the server so we should be good now